css/components/Navigation/navigation.css

```diff
@@ -17,3 +27,101 @@
   vertical-align: middle;
   background: var(--COLOR-RED-500);
 }
+
+.navbar {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 2rem;
+}
+
+.navbar-align {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
```

Comment:
same here repetitive code, can be avoided by using flex as helper class

Reply:
Thanks, Shekhar for pointing this out.